ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/lvm.py

```diff
@@ -369,14 +369,12 @@ def makeLVMap(self, columns):
             lv_om.modname = 'ZenPacks.zenoss.LinuxMonitor.SnapshotVolume'
         elif columns['lv_metadata_size']:
             lv_om.id = 'tp-{}'.format(self.prepId(columns['vg_name'])+'_'+self.prepId(columns['lv_name']))
-            lv_om.metadatasize = columns['lv_metadata_size']
+            lv_om.metadatasize = int(columns['lv_metadata_size'])
```

Comment:
This change causes a unit test failure. The unit test should probably just be updated to expect an int.